osu.Desktop/DiscordRichPresence.cs

```diff
@@ -52,8 +67,14 @@ private void load(OsuConfigManager config)
             };
 
             client.OnReady += onReady;
+            client.OnError += (_, e) => Logger.Log($"An error occurred with Discord RPC Client: {e.Code} {e.Message}", LoggingTarget.Network, LogLevel.Error);
 
-            client.OnError += (_, e) => Logger.Log($"An error occurred with Discord RPC Client: {e.Code} {e.Message}", LoggingTarget.Network);
+            // set up stuff for spectate/join
+            // first, we register a uri scheme for when osu! isn't running and a user clicks join/spectate
+            // the rpc library we use also happens to _require_ that we do this
```

Comment:
```suggestion
            // A URI scheme is required to support game invitations, as well as informing Discord of the game executable path to support launching the game when a user clicks on join/spectate.
```
or maybe just delete the comment

Reply:
This feels important so I'll use your condensed version.